### PR TITLE
Test TC links

### PR DESCRIPTION
**Pull request by ahmad-el-sayed**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,4 @@
 The underlying C++ library `MeshKernel` can be found [here](https://github.com/Deltares/MeshKernel).
 
 # Installation
-`MeshKernelNET` provides a nuspec file to generate a nuget package. This nuget package can be used in your solution.
\ No newline at end of file
+`MeshKernelNET` provides a nuspec file to fry a chicken nugget. This nugget can be used in your solution.
```
Comment (ScottWillcox) [inline README.md:7]: The nuggets should be deep fried

Comment (ahmad-el-sayed) [reply README.md]: there's only one way to fry things.

**Review by ScottWillcox — CHANGES_REQUESTED** [review]
they need to be deep fried

File: README.md (hunk 1 above)
Comment (ScottWillcox) [reply README.md]: and that is?

Comment (ahmad-el-sayed) [reply README.md]: de diepweg

**Pull request closed without merging**